spec/models/integration_collection_mappings/netsuite_collection_mapping_spec.rb

```diff
@@ -6,7 +6,7 @@
   subject(:mapping) { build(:netsuite_collection_mapping) }
 
   let(:mapping_types) do
-    %i[fallback_item coupon subscription_fee minimum_commitment tax prepaid_credit credit_note]
+    %i[fallback_item coupon subscription_fee minimum_commitment tax prepaid_credit credit_note account]
```

Comment:
why did you change `netsuite` mapping types? as the feature is about adding a new integration for Xero I wouldn't expect other integrations to change.

Reply:
Because this example fails otherwise https://github.com/getlago/lago-api/pull/2160/files/3295acbb1ddf92507a094d78246577df3db169ad#diff-5be09cab6e233e232c33fa3e68fc77d9d3d7f266d41062841f0505d3a83b8f3eL12

I'm moving it to spec/models/integration_collection_mappings/base_collection_mapping_spec.rb